packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -153,7 +156,9 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   }
 
   async setTopic(topicName: string): Promise<void> {
-    await this.handlers.updateThreadTopicName(topicName);
+    await this.asyncTeeErrorToEventEmitter(async () => {
+      await this.handlers.updateThreadTopicName(topicName);
```

Comment:
Will we add error wrapper to updateTopic function in stateful client? If yes, then we might not need to wrap it again in adapter implementation